src/category_theory/monad/equiv_mon.lean

```diff
@@ -87,27 +87,42 @@ def Mon_to_Monad : Mon_ (C ⥤ C) ⥤ Monad C :=
       finish,
     end,
     ..f.hom } }
+
+namespace Monad_Mon_equiv
```

Comment:
The changes in this file are unrelated to the PR. It seems the definitions here were teetering at the end of our `-T100000` limit, and adding a few more simp lemmas tipped them over. I split the affected declaration into three pieces, and took the opportunity to make the naming more boring.